processing/src/test/java/org/apache/druid/segment/join/table/RowBasedIndexedTableTest.java

```diff
@@ -179,4 +179,13 @@ public void testVersion()
     Assert.assertEquals(JoinTestHelper.INDEXED_TABLE_VERSION, countriesTable.version());
     Assert.assertEquals(JoinTestHelper.INDEXED_TABLE_VERSION, regionsTable.version());
   }
+
+  @Test
+  public void testIsCacheable() throws IOException
+  {
+    Assert.assertFalse(countriesTable.isCacheable());
+    RowBasedIndexedTable<Map<String, Object>> countriesTableWithCacheKey = JoinTestHelper.createCountriesIndexedTableWithCacheKey();
+    Assert.assertTrue(countriesTableWithCacheKey.isCacheable());
+    Assert.assertEquals(countriesTableWithCacheKey.computeCacheKey(), JoinTestHelper.INDEXED_TABLE_CACHE_KEY);
```

Comment:
Please use `Assert.assertArrayEquals()`. Also the expected result should be the first argument.